references/classification/README.md

```diff
@@ -19,21 +19,19 @@ pip install pyronear
 
 ## Train
 
-### Wildfire
-
-Download Dataset from https://drive.google.com/file/d/1Y5IyBLA5xDMS1rBdVs-hsVNGQF3djaR1/view?usp=sharing
+### OpenFire
 
-This dataset is protected by a password, please contact us at contact@pyronear.org
+You can also use out open-source dataset without password
```

Comment:
I think you forgot part of my previous comment : 
"out open-source ..." --> "**our** open-source ..."

Reply:
And "without **a** password"

Reply:
:)